src/LineageTree/lineageTree.py

```diff
@@ -2018,7 +2018,8 @@ def get_ancestor_at_t(self, n: int, time: int = None):
             ancestor = self.predecessor.get(ancestor, [-1])[0]
         return ancestor
 
-    def get_simple_tree(self, r: int, time_resolution: int = 1) -> tuple:
+    
+    def get_simple_tree(self, r: int, time: int = 500, time_resolution: int = 1) -> tuple:
```

Comment:
The `time` parameter is not explicit enough.

I am not sure what it is you'd like to do here tbh.

Reply:
Also you should add `time` description in the docstring